2017-05-26 16:14:15,720 - INFO - [Elapsed time: 0.00] Creating history 8861
2017-05-26 16:14:15,795 - INFO - [Elapsed time: 0.08] d33e32db742aed56
2017-05-26 16:14:15,795 - INFO - [Elapsed time: 0.08] Uploading files to galaxy...
2017-05-26 16:14:15,795 - INFO - [Elapsed time: 0.08] Finding files on the NAS...
2017-05-26 16:14:16,520 - INFO - [Elapsed time: 0.80] Uploading files from the NAS...
2017-05-26 16:14:16,520 - INFO - [Elapsed time: 0.80] 1 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0985_S29_L001_R1_001.fastq.gz
2017-05-26 16:14:16,520 - INFO - [Elapsed time: 0.80] Uploading...
2017-05-26 16:14:21,373 - INFO - [Elapsed time: 5.65] 2 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0985_S29_L001_R2_001.fastq.gz
2017-05-26 16:14:21,373 - INFO - [Elapsed time: 5.65] Uploading...
2017-05-26 16:14:45,802 - INFO - [Elapsed time: 30.08] 3 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0984_S28_L001_R1_001.fastq.gz
2017-05-26 16:14:45,802 - INFO - [Elapsed time: 30.08] Uploading...
2017-05-26 16:15:20,246 - INFO - [Elapsed time: 64.53] 4 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0984_S28_L001_R2_001.fastq.gz
2017-05-26 16:15:20,246 - INFO - [Elapsed time: 64.53] Uploading...
2017-05-26 16:15:59,416 - INFO - [Elapsed time: 103.70] 5 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0958_S2_L001_R1_001.fastq.gz
2017-05-26 16:15:59,417 - INFO - [Elapsed time: 103.70] Uploading...
2017-05-26 16:17:17,068 - INFO - [Elapsed time: 181.35] 6 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0958_S2_L001_R2_001.fastq.gz
2017-05-26 16:17:17,069 - INFO - [Elapsed time: 181.35] Uploading...
2017-05-26 16:17:47,517 - INFO - [Elapsed time: 211.80] 7 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0961_S5_L001_R1_001.fastq.gz
2017-05-26 16:17:47,517 - INFO - [Elapsed time: 211.80] Uploading...
2017-05-26 16:19:31,554 - INFO - [Elapsed time: 315.84] 8 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0961_S5_L001_R2_001.fastq.gz
2017-05-26 16:19:31,554 - INFO - [Elapsed time: 315.84] Uploading...
2017-05-26 16:19:55,323 - INFO - [Elapsed time: 339.60] 9 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0962_S6_L001_R1_001.fastq.gz
2017-05-26 16:19:55,323 - INFO - [Elapsed time: 339.60] Uploading...
2017-05-26 16:20:36,088 - INFO - [Elapsed time: 380.37] 10 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0962_S6_L001_R2_001.fastq.gz
2017-05-26 16:20:36,088 - INFO - [Elapsed time: 380.37] Uploading...
2017-05-26 16:20:53,958 - INFO - [Elapsed time: 398.24] 11 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0960_S4_L001_R1_001.fastq.gz
2017-05-26 16:20:53,959 - INFO - [Elapsed time: 398.24] Uploading...
2017-05-26 16:21:05,486 - INFO - [Elapsed time: 409.77] 12 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0960_S4_L001_R2_001.fastq.gz
2017-05-26 16:21:05,486 - INFO - [Elapsed time: 409.77] Uploading...
2017-05-26 16:22:06,531 - INFO - [Elapsed time: 470.81] 13 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0975_S19_L001_R1_001.fastq.gz
2017-05-26 16:22:06,531 - INFO - [Elapsed time: 470.81] Uploading...
2017-05-26 16:22:09,986 - INFO - [Elapsed time: 474.27] 14 of 15: Uploading /mnt/nas/MiSeq_Backup/150203/2014-SEQ-0975_S19_L001_R2_001.fastq.gz
2017-05-26 16:22:09,986 - INFO - [Elapsed time: 474.27] Uploading...
2017-05-26 16:22:27,172 - INFO - [Elapsed time: 491.45] 15 of 15: Uploading /mnt/nas/WGSspades/150203_Assembled/BestAssemblies/2014-SEQ-0985.fasta
2017-05-26 16:22:27,172 - INFO - [Elapsed time: 491.45] Uploading...
2017-05-26 16:22:54,265 - INFO - [Elapsed time: 518.55] Waiting for files to finish uploading...
2017-05-26 16:23:14,457 - INFO - [Elapsed time: 538.74] Finished uploading.
2017-05-26 16:23:14,457 - INFO - [Elapsed time: 538.74] Building list of dataset pairs...
2017-05-26 17:20:50,586 - INFO - [Elapsed time: 3994.87] [Error Dump]
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 377, in _make_request
    httplib_response = conn.getresponse(buffering=True)
TypeError: getresponse() got an unexpected keyword argument 'buffering'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/devon/PycharmProjects/auto-SNVPhyl/main.py", line 52, in run
    return self.main()  # Return the path to the results zip
  File "/home/devon/PycharmProjects/auto-SNVPhyl/main.py", line 155, in main
    self.build_list()
  File "/home/devon/PycharmProjects/auto-SNVPhyl/main.py", line 473, in build_list
    elements = [
  File "/usr/local/lib/python3.5/dist-packages/bioblend/galaxy/histories/__init__.py", line 394, in create_dataset_collection
    return Client._post(self, payload, id=history_id, contents=True)
  File "/usr/local/lib/python3.5/dist-packages/bioblend/galaxy/client.py", line 157, in _post
    files_attached=files_attached)
  File "/usr/local/lib/python3.5/dist-packages/bioblend/galaxyclient.py", line 125, in make_post_request
    verify=self.verify, params=post_params)
  File "/usr/lib/python3/dist-packages/requests/api.py", line 107, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/api.py", line 53, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 468, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 576, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 376, in send
    timeout=timeout
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 560, in urlopen
    body=body, headers=headers)
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 379, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/lib/python3.5/http/client.py", line 1197, in getresponse
    response.begin()
  File "/usr/lib/python3.5/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.5/http/client.py", line 258, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/lib/python3.5/socket.py", line 575, in readinto
    return self._sock.recv_into(b)
KeyboardInterrupt

